### PR TITLE
Javascript

### PR DESCRIPTION
**Pull request by willm**
Web client to native client example

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -130,7 +133,7 @@ set_target_properties(datachannel-tests PROPERTIES
 	CXX_STANDARD 17)
 set_target_properties(datachannel-tests PROPERTIES OUTPUT_NAME tests)
 target_include_directories(datachannel-tests PRIVATE ${CMAKE_CURRENT_SOURCE_DIR}/src)
-target_link_libraries(datachannel-tests datachannel)
+target_link_libraries(datachannel-tests datachannel nlohmann_json::nlohmann_json)
```
Comment (paullouisageneau) [inline CMakeLists.txt:136]: This might not be necessary since `nlohmann/json` is header-only.

File: include/rtc/peerconnection.hpp [hunk 2]
```diff
@@ -87,6 +87,8 @@ class PeerConnection : public std::enable_shared_from_this<PeerConnection> {
 	void onStateChange(std::function<void(State state)> callback);
 	void onGatheringStateChange(std::function<void(GatheringState state)> callback);
 
+	std::string connectionInfo;
```
Comment (paullouisageneau) [inline include/rtc/peerconnection.hpp:90]: This should be kept in the test itself.

**Comment by paullouisageneau** [comment]
OK, I'll merge this in `dev` first to do some cleanup and plumbing!

File: CMakeLists.txt (hunk 1 above)
Comment (willm) [reply CMakeLists.txt:136]: I don't think this lib is used at all, so it can actually be removed

**Pull request merged**